src/shogun/machine/IterativeMachine.h

```diff
@@ -109,7 +116,17 @@ namespace shogun
 
 		/** To be overloaded in subclasses to initialize the model for training
 		  */
-		virtual void init_model(CFeatures* data = NULL) = 0;
+		virtual void init_model(CFeatures* data = NULL)
```

Comment:
@shubham808 you should also review/discuss this as some of your classes are changed (better API)

Reply:
yeah init_model is taking features and labels as arguments which is more flexible :+1: